### ISSUE TITLE
Won't the "use client" inside CreateNote expose the pocketbase endpoint to the client/public

### ISSUE THREAD

**Issue by themegabyte** [fireship-io/next13-pocketbase-demo#4]
Hello Jeff! 

Thank you for all the hard work you have put in your videos. They are my go to for leasure and to learn something new.

IN your latest one: https://www.youtube.com/watch?v=__mSgDEOyv8

I noticed that when you create a note you make a fetch request from the client, I am correct to assume this URL will be exposed to the public?

Link to the line: 
https://github.com/fireship-io/next13-pocketbase-demo/blob/main/app/notes/CreateNote.tsx#L20

Thanks again to Jeff (and to all the people more smarter then me).

**Comment by ERmilburn02** [comment]
Yes, it would, if it wasn't for the fact that Jeff forgot to prevent the default behaviour of the form, causing it to refresh before it sends the request.
